data/pirate jobs.txt

```diff
@@ -2581,6 +2581,362 @@ mission "Cargo Theft (South)"
 		outfit "Secret Cargo" -1
 		payment 1500000
 		dialog "The pirates of <planet> take the secret cargo from your cargo hold and pay you <payment>."
+		
+mission "FW Assasination [1]"
+	name "Assassinate a political target"
+	description "A key polititican is out on a pleasure flight in their ship, the <npc>. Find them and destroy them."
+	repeat
+	job
+	to offer
+		"combat rating" > 200
+	source
+		government "Pirate"
+		attributes "south pirate"
+	npc kill
+		government "Free Worlds"
+		personality heroic staying target
+		system
+			distance 1 3
+		fleet
+			names civilian
+			variant
+				"Flivver"
+		dialog "<npc> has been destroyed, and it's pilot killed. Return to <destination> for you payment."
+	on complete
+		payment 350000
+		dialog "You land at <planet> and the local bounty office transfers the payment of <payment>."
+		reputation "Free Worlds" -= 20
+		
+mission "FW Assasination [2]"
+	name "Assassinate a political target"
+	description "A key political target is gathering an escort of personal ships nearby. Destroy them and whatever fleet ships they've gathered."
+	repeat
+	job
+	to offer
+		"combat rating" > 400
+	source
+		government "Pirate"
+		attributes "south pirate"
+	npc kill
+		government "Free Worlds"
+		personality staying uninterested target
+		system
+			distance 1 3
```

Comment:
I doubt that any important individual is going to sit right outside a pirate system; I think that 2 should be the absolute minimum in any of these cases, perhaps even 3. Also, don't be afraid to increase the maximum distance to something like 5. That might be a lot of looking that a player needs to do, but they're already taking a job with a giant penalty, so that coupled with a high payment would justify it.